resources/charlie/config-example.toml

```diff
@@ -0,0 +1,166 @@
+# ================================
+# Configuration options for a node
+# ================================
+[node]
+
+# Path (absolute, or relative to this config.toml) to the chainspec configuration file.
+chainspec_config_path = '/etc/casper/chainspec.toml'
+
+# The maximum number of deploys permitted in a single block.
+block_max_deploy_count = 3
+# If set, use this hash as a trust anchor when joining an existing network.
+# trusted_hash =
+
+
+# =================================
+# Configuration options for logging
+# =================================
+[logging]
+
+# Abbreviate module names in text ouptut.
+abbreviate_modules = false
+
+# Output format. Change this to JSON if you want to parse your logs.
+format = "text"
+
+
+# ====================================
+# Configuration options for consensus
+# ====================================
+[consensus]
+
+# Path (absolute, or relative to this config.toml) to validator's secret key file used to sign
+# consensus messages.
+secret_key_path = '/etc/casper/validator_keys/secret_key.pem'
+
+
+# ====================================
+# Configuration options for networking
+# ====================================
+[network]
+
+# The public address of the node.
+#
+# It must be publicly available in order to allow peers to connect to this node.
+# If the port is set to 0, the actual bound port will be substituted.
+public_address = '[IP ADDRESS]:0'
+
+# Address to bind to for listening.
+# If port is set to 0, a random port will be used.
+bind_address = '0.0.0.0:34553'
+
+# Addresses to connect to in order to join the network.
+#
+# If not set, this node will not be able to attempt to connect to the network.  Instead it will
+# depend upon peers connecting to it.  This is normally only useful for the first node of the
+# network.
+#
+# Multiple addresses can be given and the node will attempt to connect to each, requiring at least
+# one connection.
+known_addresses = ['54.177.84.9:34553','18.144.69.216:34553','13.57.251.65:34553']
+
+# The interval (in milliseconds) between each fresh round of gossiping the node's public address.
+gossip_interval = 30000
+
+
+# =============================================
+# Configuration options for the HTTP API server
+# =============================================
+[http_server]
+
+# Interface to bind to.  Defaults to loopback address.
+bind_interface = '127.0.0.1'
+
+# Port to bind to.  Use 0 for a random port.
+bind_port = 7777
+
+
+# ===============================================
+# Configuration options for the storage component
+# ===============================================
+[storage]
+
+# Optional path (absolute, or relative to this config.toml) to the folder where any files created
+# or read by the storage component will exist.
+#
+# If the folder doesn't exist, it and any required parents will be created.
+#
+# If unset, defaults to:
```

Comment:
These defaults should be updated to reflect that the node will run as root.  I guess the Windows entry can be removed (I was a bit optimistic including that one!)